Backend/src/main/java/ca/mcgill/cooperator/controller/NotificationController.java

```diff
@@ -43,6 +43,20 @@ public NotificationDto getNotificationById(@RequestParam int id) {
 
         return ControllerUtils.convertToDto(n);
     }
+    
+
+    /**
+     * Sets notification seen by id
+     *
+     * @param id
+     * @return NotificationDto
+     */
+    @GetMapping("/seen/{id}")
```

Comment:
Should be a PUT request since it's updating the notification. I would change it to `/{id}/seen` so that it's more obvious what the ID is for